extensions/mssql/package.json

```diff
@@ -383,6 +383,11 @@
           "type": "boolean",
           "description": "%mssql.ignorePlatformWarning%",
           "default": false
+        },
+        "mssql.useNativeCredentials": {
+          "type": "boolean",
+          "description": "%mssql.useNativeCredentials%",
+          "default": false
```

Comment:
Given that this was brought up as a security issue would it make more sense to have it default to try and then just ask (and strongly suggest) that the user migrate on launch, but allow them to specifically opt-out if they wish?

(TBH though I'm also in favor of just migrating the secrets regardless, where the secrets are stored shouldn't be something the user needs to know or care about)